src/test/cypress/e2e/exam/ExamAssessment.cy.ts

```diff
@@ -102,101 +95,125 @@ describe('Exam assessment', () => {
 
         it('Assess a programming exercise submission (MANUAL)', () => {
             cy.login(tutor, '/course-management/' + course.id + '/exams');
-            examManagement.openAssessmentDashboard(exam.id!, examDuration);
+            examManagement.openAssessmentDashboard(exam.id!, 155000);
             startAssessing();
             examAssessment.addNewFeedback(2, 'Good job');
             examAssessment.submit();
             cy.login(student, '/courses/' + course.id + '/exams/' + exam.id);
             editorPage.getResultScore().should('contain.text', '66.2%, 6 of 13 passed, 6.6 points').and('be.visible');
+            programmingAssessmentSuccessful = true;
+        });
+
+        it('Complaints about programming exercises assessment', () => {
```

Comment:
Is the order of test guaranteed? At least in Unit you have to specifically add @Order for such things

Reply:
I could not find anything about the test order within the official documentation except for this part:
https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#The-order-of-hook-and-test-execution-is-as-follows

I found this answer in a github issue, that hinted, that the tests run in the given order:
https://github.com/cypress-io/cypress/issues/2901#issuecomment-444422344